src/Farama_Notifications/notices.py

```diff
@@ -1 +1,5 @@
-notices = dict()
+# notifications are a dict of dicts
+notifications = dict()
+notifications["gymnasium"] = dict()
+
+notifications["gymnasium"]["69.420"] = "This version does not exist."
```

Comment:
Would you want to use a testing repo rather than a made up version number

Reply:
This line is added just so I/others can see what adding messages to versions should look like.

Reply:
Could you do it sa a comment rather than an actual entry with the following comment

``` python
# notification["project"]["version"] = "Notification message" 
```